CODEOWNERS

```diff
@@ -1,23 +1,30 @@
 # Go engineers are automatically added as reviewers when changes are made to go
+
 # files or related backend files.
-*.go @fleetdm/go
+
+\*.go @fleetdm/go
```

Comment:
github is flagging this and the other edits below as invalid due to the `\` at the beginning, was it added by accident?

Reply:
Yes definitely accident!